src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,19 @@
+const ArticleCard = ({ article }) => {
+  const date = new Date(article.created_at).toLocaleDateString();
+  return (
+    <>
+      <ul className="article-card">
+        <p>{article.topic}</p>
+        <p>{date}</p>
+        <h3>{article.title}</h3>
+        <p>{article.author}</p>
+        <img src={article.article_img_url}></img>
+        <p>Votes ({article.votes})</p>
+        <p>Comments ({article.comment_count})</p>
+        <button>Read</button>
```

Comment:
Would be nice to see some more info as a user. I'm looking at your article cards and just see the word, cooking and grumpy19. Lets add some context like Author: grumpy19 and Topic: cooking etc